### PR TITLE
Add Section Leaders to !groups

### PR DESCRIPTION
**Pull request by AeonicX7**
also minor grammatical edits

### CODE REVIEW

File: server/chat-commands/info.ts [hunk 1]
```diff
@@ -1539,7 +1539,8 @@ export const commands: Chat.ChatCommands = {
 		const globalRanks = [
 			`<strong>Global ranks</strong>`,
 			`+ <strong>Global Voice</strong> - They can use ! commands like !groups`,
-			`% <strong>Global Driver</strong> - The above, and they can also lock users and check for alts`,
+			`§ <strong>Section Leader</strong> - They oversee rooms in a particular section`,
+			`% <strong>Global Driver</strong> - Like Voice, and they can lock users and check for alts`,
```
Comment (AnnikaCodes) [inline server/chat-commands/info.ts:1543]: You'll need to update the Driver translation texts too.

Comment (AeonicX7) [reply server/chat-commands/info.ts:1543]: Agh, okay. I'll post in wail about it.

File: translations/portuguese/main.ts [hunk 2]
```diff
@@ -16,7 +16,8 @@ export const translations: Translations = {
 
 		"<strong>Global ranks</strong>": "<strong>Cargos Globais</strong>",
 		"+ <strong>Global Voice</strong> - They can use ! commands like !groups": "+ <strong>Global Voice</strong> - Eles podem usar comandos com !, tal como !groups",
-		"% <strong>Global Driver</strong> - The above, and they can also lock users and check for alts": "% <strong>Global Driver</strong> - O de cima, além de poderem dar locks e verificar contas alternativas",
+		"§ <strong>Section Leader</strong> - They oversee rooms in a particular section": "§ <strong>Section Leader</strong> - Eles supervisionam salas de uma seção específica",
```
Comment (Zarel) [inline translations/portuguese/main.ts:19]:
```suggestion
		"§ <strong>Section Leader</strong> - They oversee rooms in a particular section": "§ <strong>Section Leader</strong> - Eles supervisionam salas de uma seção específica",
```

**Comment by Zarel** [comment]
I'm assuming it's not just the trailing space, we're still waiting on other translations? Let's draft this for now, then.

File: translations/spanish/main.ts [hunk 3]
```diff
@@ -16,7 +16,8 @@ export const translations: Translations = {
 
 		"<strong>Global ranks</strong>": "<strong>Rangos Globales</strong>",
 		"+ <strong>Global Voice</strong> - They can use ! commands like !groups": "+ <strong>Global Voice</strong> - Pueden utilizar comandos con ! como !groups",
-		"% <strong>Global Driver</strong> - The above, and they can also lock users and check for alts": "% <strong>Global Driver</strong> - Lo mismo que arriba y también pueden dar locks y revisar las alts",
+		"§ <strong>Section Leader</strong> - They oversee rooms in a particular section": "§ <strong>Section Leader</strong> - Supervisan las salas de una sección en particular"
```
Comment (Zarel) [inline translations/spanish/main.ts:19]:
```suggestion
		"§ <strong>Section Leader</strong> - They oversee rooms in a particular section": "§ <strong>Section Leader</strong> - Supervisan las salas de una sección en particular",
```

**Pull request merged**